rle/rleplus_reader.go

```diff
@@ -76,6 +77,47 @@ func DecodeRLE(buf []byte) (RunIterator, error) {
 	return it, nil
 }
 
+func ValidateRLE(buf []byte) error {
+	if len(buf) > 0 && buf[len(buf)-1] == 0 {
+		// trailing zeros bytes not allowed.
+		return xerrors.Errorf("not minimally encoded: %w", ErrDecode)
+	}
+	bv := readBitvec(buf)
+
+	ver := bv.Get(2) // Read version
+	if ver != Version {
+		return ErrWrongVersion
+	}
+	bv.Get(1)
```

Comment:
Nit: do we even allow this bit to be anything but 1? I don't think we can fix this now, but I think we may want to note it.

Also, let's leave a comment on why we're doing this.

Reply:
Yes, it the value of the first run. So if it is 1, the first run will be set bits, otherwise it will be a run of unset bits.

Reply:
Ah, wait, nvm. You're right.